### PR TITLE
Mock/tooltip with test description

### PR DESCRIPTION
**Pull request by eng-esther**

### CODE REVIEW

File: src/components/CompareResults/CompareResultsTable.tsx [hunk 1]
```diff
@@ -14,7 +14,6 @@ import CompareResultsTableRow from './CompareResultsTableRow';
 const tableHeaders: CompareResultsTableHeaders[] = [
   { id: 'platform', label: 'Platform', align: 'center' },
   { id: 'graph', label: 'Graph', align: 'center' },
-  { id: 'suite', label: 'Suite', align: 'left' },
```
Comment (eng-esther) [inline src/components/CompareResults/CompareResultsTable.tsx:17]: I didnt make any change here maybe it's the diff between staging and mocking ?

**Comment by julienw** [comment]
I don't know what's happening with this PR really, so I'll just remove the r? request :) If you want me to review this PR please give me some more context. Thanks :-)

**Comment by eng-esther** [comment]
> I don't know what's happening with this PR really, so I'll just remove the r? request :) If you want me to review this PR please give me some more context. Thanks :-)

Hey, here's some context for this
https://mozilla-hub.atlassian.net/browse/FXP-2405

**Comment by kimberlythegeek** [comment]
@julienw rather than fully implementing new features, we have decided to "mock" certain features we would like to include in our demo for All Hands, purely for the purpose of displaying them in the demo and/or lightning talk

File: src/components/CompareResults/CompareResultsTable.tsx (hunk 1 above)
Comment (kimberlythegeek) [reply src/components/CompareResults/CompareResultsTable.tsx:17]: yes, likely so. I'll create a patch to update the mocking branch to match the latest staging.

**Review by kimberlythegeek — APPROVED** [review]

**Comment by kimberlythegeek** [comment]
I've merged #234 to update the mocking branch. Please rebase and see if it resolves the file changes that you did not make.

**Comment by eng-esther** [comment]
> I've merged #234 to update the mocking branch. Please rebase and see if it resolves the file changes that you did not make.

issue seems fixed/ files fixed

**Pull request merged**